pkgs/servers/web-apps/snipe-it/default.nix

```diff
@@ -7,12 +7,19 @@ let
     noDev = true; # Disable development dependencies
   }).overrideAttrs (attrs : {
     installPhase = attrs.installPhase + ''
+      # Before symlinking the following directories, copy the invalid_barcode.gif
+      # to a different location. The `snipe-it-setup` oneshot service will then
+      # copy the file back during bootstrap.
+      cp $out/public/uploads/barcodes/invalid_barcode.gif $out/
```

Comment:
```suggestion
      cp $out/public/uploads/barcodes/invalid_barcode.gif{,.backup}
```
That would be a bit nicer than just dumping it into $out

Reply:
We symlink the public/ directory with the dataDir so this likely won't work.

Reply:
Then maybe some other directory except plain $out? $out/share/snipe-it/invalid_barcode.gif

Reply:
Alright, let's do it that way.